src/main/scala/nl/knaw/dans/easy/fedoratobag/EasyFedoraToBagApp.scala

```diff
@@ -161,7 +161,7 @@ class EasyFedoraToBagApp(configuration: Configuration) extends DebugEnhancedLogg
         }
     }
 
-    def hasTooManyFiles(selectedForSecondBag: List[FileInfo], selectedForFirstBag: List[FileInfo]) = {
+    def numberOfFilesBelowCutoff(selectedForSecondBag: List[FileInfo], selectedForFirstBag: List[FileInfo]) = {
```

Comment:
Getting rid of the negatives might be better (don't forget where called):

      if (selectedForFirstBag.size > options.cutoff || selectedForSecondBag.size > options.cutoff) {
        logger.warn(s"too many files ${ selectedForFirstBag.size }, ${ selectedForSecondBag.size }")
        true
      }
      else options.noPayload

The case class `DatasetInfo` may have a better naming convention: `withPayload`